astroid/interpreter/_import/util.py

```diff
@@ -72,7 +72,8 @@ def is_namespace(modname: str) -> bool:
         if found_spec and found_spec.submodule_search_locations:
             last_submodule_search_locations = found_spec.submodule_search_locations
 
-    if found_spec is None:
-        return False
-
-    return found_spec.origin is None
+    return (
+        found_spec is not None
```

Comment:
Should we use `last_submodule_search_locations` here?

Reply:
I don't know, I suspect (without looking too closely) that could create false positives if a portion of the path was a valid namespace package but the rest of the path wasn't. That variable is really just a hack to get around the KeyError described in the comment, I wouldn't trust it to represent more than that.